audit-service/src/unit-tests/java/uk/gov/gchq/palisade/service/audit/ResponseEntityAssert.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2018-2021 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.palisade.service.audit;
+
+import org.assertj.core.api.AbstractAssert;
+import org.assertj.core.api.AbstractStringAssert;
+import org.assertj.core.api.ObjectAssert;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Assertion methods for {@code ResponseEntity}s.
+ *
+ * @param <T> The type contained in the response entity
+ */
+public class ResponseEntityAssert<T> extends AbstractAssert<ResponseEntityAssert<T>, ResponseEntity<T>> {
```

Comment:
most of these are unused or only used once, why create a library for them when we can do all functionality with native assertJ?

Reply:
I got into the habit of creating these at the point the first test was written. As the IDE pulls in the static import after typing `assertThat`, the closest matching assertion is selected. So over time, they start getting used more and more. The assertions become nice to read. But at the start, yes, just a single test. I think the problem in this case is that the tests have already been written.